local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -135,6 +135,21 @@
           kind: documentation
           dependencies: ["https://github.com/DataDog/puppet-datadog-agent/blob/master/README.md"]
 
+  - repo_name: datadog-formula
+    contents:
+
+    - action: pull-and-push-file
+      branch: ruth/docs-632
```

```suggestion
      branch: master
```
Change before merging